src/esp/assets/managers/SceneAttributesManager.h

```diff
@@ -57,9 +85,37 @@ class SceneAttributesManager
   }  // SceneAttributesManager::setSceneValsFromPhysicsAttributes
 
  protected:
+  /**
+   * @brief Scene is file-based, described by @ref sceneFilename; populate a
+   * returned scene attributes with appropriate data.
+   *
+   * @param sceneFilename The configuration file to parse.
+   * @param registerTemplate whether to add this template to the library or not.
+   * @return a reference to the physics simulation meta data object parsed from
+   * the specified configuration file, or nullptr if fails.
+   */
```

Comment:
Clean-up this comment and reference the intent to use this function for backward compatibility with `AssetInfo::fromPath`.